cmd/podman/system/reset.go

```diff
@@ -65,6 +65,34 @@ Are you sure you want to continue? [y/N] `)
 		}
 	}
 
+	// Get all the external containers in use
+	listCtn, _ := registry.ContainerEngine().ContainerListExternal(registry.Context())
+	listCtnIds := make([]string, 0, len(listCtn))
+	for _, externalCtn := range listCtn {
+		listCtnIds = append(listCtnIds, externalCtn.ID)
```

Comment:
Please specify Names or both.  These IDs mean very little to Human beings, where as they might mean more if they saw:

alpine-working-container

Reply:
@rhatdan Added both of them but this might change as @vrothberg has a better suggestion as being discussed below.
```bash 
WARNING! Following External Containers will be purged
CONTAINER ID  NAME
c7d50cbfff74  busybox-working-container
```